_episodes/03-github-pages.md

```diff
@@ -3,275 +3,126 @@ title: "Hosting Pages and Websites on GitHub"
-hosting service by GitHub that takes files from your GitHub repository that is configured as a website,
-optionally runs the files through a build process, and publishes them as a website or a web page.
+hosting service by GitHub that takes files (Markdown, HTML, CSS, JavaScript, etc.) 
+from your GitHub repository which is configured as a website,
+optionally runs the files through a build process, combines them and publishes them as a website or a Web page.
 Any changes you do to the files in your website's GitHub repository
 will be rendered live in the website.
 
-# Publishing a repository as a website
-
-GitHub Pages is a service from GitHub to publish a website directly from a Git repository.
-
-> ## Why Github?
->
-> There are other services available to create and publish website but one of the main advantage of GitHub Pages is that you can version control your website and therefore keep track of all your changes. This is particularly helpful for collaborating on a project website. [GitLab](https://about.gitlab.com/) offers very similar services but GitHub pages is the simplest approach.
->
-{: .callout}
-
-Let's start from the github repository we have created in the previous episode {{ site.project-pages }}. One important file you should have is `index.md`: it will become the homepage of your project website.
+There are other services available to create and publish website but one of the main advantages of GitHub Pages is that you can version control your website and therefore keep track of all your changes. This is particularly helpful for collaborating on a project website. [GitLab](https://about.gitlab.com/) offers very similar services but GitHub pages is the simplest approach.
 
-In this episode, you will learn how to turn your project repository into a website.
+Let's continue from the GitHub repository we have created in the previous episode. One important file you should have is `index.md`: it will become the homepage of your project website.
 
-## Turn on GitHub Pages for your main branch
+## Turning on GitHub Pages for Your Repository
 
-GitHub Pages uses a special branch in your GitHub repository to look for website content, and by default this branch is named `gh-pages`. 
+There are two approaches to tell GitHub your repository is hosting a website and configure GitHub pages to render it. 
 
-It is a convention used by Github and we need to change this default behavior because the content of our repository is under a branch called *main*.
+The first one is to create a new branch called `gh-pages`. By convention, GitHub Pages uses this branch to look for the website content and by creating it you implicitly tell GitHub that you want your content published. Once you can create `gh-pages` from your current branch (`main`, created by default when you created the repository), you can then choose to delete the branch `main` to avoid any confusion. 
```

Comment:
I wonder if it makes sense to talk about gh-pages at this stage. 

Wouldn't be simpler to show how to select `main` branch to "create" a website and than have a called out about gh-pages.

Reply:
Yes, might have got things backward - I tell them about gh-pages and then go and explain the approach which is actually less favourable. I'll fix this tomorrow as per your suggestion @annefou.

Reply:
OK, I fixed the text to address this comment @annefou via ec981b4891081876fffcc766ddc2ba1439c0226b.